jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+---
+title: Python Developers (x2 roles)
+company: SportPursuit
+url: https://www.sportpursuit.com/
+location: London, UK
+
+contract: Permanent
+contact:
+    name: Steve Blake
+    email: join@sportpursuit.co.uk
+    phone: # Phone number (optional)
+    appliction form: https://sportpursuit.peoplehr.net/Pages/JobBoard/Opening.aspx?v=3b21579e-16a4-4828-941e-91ac3f4ddc5b
+created: !!timestamp 2018-04-25
+tags:
+  - python
+  - sql
+  - django
+  # include relevant tags
+---
+
+# Python Developer (x2 roles)
+
+## Overview
+SportPursuit is a leading private shopping community for over 4 million active men and women across UK and Europe. We were rated as the 5th fastest growing business in the UK (The Times Sage Startup Track 15 2015) and offer a culture where self-starters have an opportunity to have a huge impact.
+
+Our mission is to offer amazing offers on the best sports products and services from the world’s leading sports brands to our members in week long ‘flash’ sales. Our deals are only available to our members behind our sign in wall; this gives our brands the confidence to deliver exclusive offers to our members whilst protecting their brands and their full price retailers from continual discounting.
+
+Our offices are located at Workspace’s flagship hub in Kennington Park next to the Oval Cricket ground and recently SportPursuit has become a member of the Future Fifty, the UK's leading programme for late-stage digital businesses.
+
+If you’re ambitious, talented, and want to shape the future of the business but don’t mind getting your hands dirty this could be just the place for you.
+
+## Summary
+We are looking for a 2 x Python Developers to assist with development on some key areas of our internal technology platform, namely our Python based ERP platform and Business Intelligence / Data Warehouse system.
```

Comment:
You should probably put the requirement nearer to the top of the job ad. Some of these technologies ought to be tags.